frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -39,6 +52,9 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
                 linkTitle={title}
                 popupTitle={popupTitle}
                 className="co-status-card__popup"
+                position={position ?? PopoverPosition.top}
```

Comment:
IMO all dashboard popovers should show on the same side.

Reply:
https://github.com/openshift/console/pull/9426#discussion_r675975024